TimberAPIExample/Examples/UIBuilderExample/UIBuilderPreviewPanel/Previews/ListViewPreview.cs

```diff
@@ -135,7 +135,7 @@ private void ExampleOnSelectionChange(IEnumerable<object> selectedItemObjects)
                 if (itemObject is not CustomSourceExample item) 
                     return;
             
-                TimberAPIExamplePlugin.Log.LogFatal(item.Text);
+                Log.LogFatal(item.Text);
```

Comment:
No idea what this method does, but this should probably be logged as error, not fatal?

Reply:
Could be changed to info